go/libraries/doltcore/cherry_pick/cherry_pick.go

```diff
@@ -106,7 +142,11 @@ func CherryPick(ctx *sql.Context, commit string, options CherryPickOptions) (str
 		return "", nil, err
 	}
 	if pendingCommit == nil {
-		return "", nil, errors.New("nothing to commit")
+		if commitProps.SkipEmpty {
+			return "", nil, nil
```

Comment:
the gDoc says that success will result in a hash (as a string). This seems to violate that. Probably worth calling out this edge case in the gDoc for the function.

Reply:
Good catch. Updated in 3ee48c2eed346a5864307e6a4aa2d7e1340b36c7